docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -54,25 +61,118 @@ This setting has precedence over each individual rule frequency. If a rule frequ
 
   - **Grafana Mimir** - use the `/prometheus` prefix. The Prometheus data source supports both Grafana Mimir and Prometheus, and Grafana expects that both the [Query API](/docs/mimir/latest/operators-guide/reference-http-api/#querier--query-frontend) and [Ruler API](/docs/mimir/latest/operators-guide/reference-http-api/#ruler) are under the same URL. You cannot provide a separate URL for the Ruler API.
 
-## Create recording rules
-
-To create recording rules, follow these steps.
+### Steps
 
 1. Click **Alerts & IRM** -> **Alerting** ->
    **Alert rules**.
 1. Select **Rule type** -> **Recording**.
-1. Click **+New recording rule**.
+1. Scroll to the **Data-source managed section** and click **+New recording rule**.
+
+#### Enter recording rule name
+
+The recording rule name must be a Prometheus metric name and contain no whitespace.
```

Comment:
We can either link out to https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels or copy their docs and mention the `[a-zA-Z_][a-zA-Z0-9_]*` regex – or too technical?